src/style.css

```
@@ -0,0 +1,75 @@
+@font-face {
+  font-family: Avenir;
+  font-weight: 300;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir;
+  font-weight: bold;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.container {
+  position: relative;
+  margin-top: 20px;
+}
+
+.default_input {
+  width: 100%;
+  border: 1px solid #e1e7ed;
+  outline: none;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  font-family: Avenir, Helvetica, sans-serif;
+  font-weight: bold;
+}
+
+.default_input::placeholder {
+  font-weight: normal;
+  color: #3d4e61;
+}
+
+.default_input:hover {
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  border-radius: 4px;
+}
+
+.default_input:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border: 1px solid #e1e7ed;
+  border-radius: 3px;
+}
+
+.big_input {
+  height: 70px;
+  font-size: 16px;
+  padding-left: 62px;
+}
+
+.small_input {
+  font-size: 14px;
+  padding-left: 33px;
+  height: 42px;
+}
+
+.big_icon {
+  position: absolute;
+  left: 26px;
+  top: 25px;
+  background-color: black;
+}
+
+.small_icon {
+  position: absolute;
+  width: 11px;
+  height: 11px;
+  left: 13px;
+  top: 15px;
+  background-color: black;
```

Why do you need this!
```suggestion
  background-color: black;
```